api/v1alpha1/edgedevice_types.go

```diff
@@ -44,6 +44,8 @@ type MetricsReceiverConfiguration struct {
 	RequestNumSamples int64  `json:"requestNumSamples,omitempty"`
 	TimeoutSeconds    int64  `json:"timeoutSeconds,omitempty"`
 	URL               string `json:"url,omitempty"`
+	// CA is a base64 encoding of CA certificate
+	CA string `json:"ca,omitempty"`
```

Comment:
Please store the CA in a secret; this way, it could be easily shared and rotated more easily.

Reply:
I do not see a reason to store it in a secret. At least not at the moment. We do not know where the CA will come from nor what kind of receiver will be installed. In general, secrets are good just for one use case. Automatic updating of files in PODs/containers. In our case there's no pod. As far as I recall, applications such as certmanager can rotate configmaps just the same.
@pkliczewski @masayag @machacekondra would appreciate your opinion as well.

Reply:
There is already a CA cert store in a secret 
https://github.com/project-flotta/flotta-operator/blob/main/pkg/mtls/caprovider_secret.go#L93
Why need to store it in edgedevice?

Reply:
I think it might be a different CA, used to create Thanos instance certificate and as such should be kept separately from device mtls CA.

Reply:
@bardielle u raise an interesting point. @jakub-dzon I think you are right. However, in other features, e.g. data upload, we provided a default implementation and allowed the user to configure his own.
@bardielle point that Jakub raised regarding configuring EdgeDevice, was that user may want different metrics server for different devices/device-groups.

Reply:
using secrets for CA seems to be common practice in k8s world